packaging/installer/UPDATE.md

```diff
@@ -20,28 +21,35 @@ icon in the local Agent dashboard's top navigation. This modal informs you wheth
 
 ## Determine which installation method you used
 
+Starting with netdata v1.33.0, you can use Netdata itself to determine the installation type by running:
+
+```bash
+netdata -W buildinfo | grep 'Install type:'
+```
+
+If this produces no output, you have an older install and will have to manually look in the Netdata config directory.
 If you are not sure where your Netdata config directory is, see the [configuration doc](/docs/configure/nodes.md). In
 most installations, this is `/etc/netdata`.
 
 Use `cd` to navigate to the Netdata config directory, then use `ls -a` to look for a file called `.install-type`.
```

Comment:
@Ferroin Line 34 contains instructions but they are stuffed away in a paragraph. 
Would you be fine to restructure the steps into an ordered list? 
I proposed the following in the blog:

<img width="747" alt="image" src="https://user-images.githubusercontent.com/13576110/154673414-8e8d4881-84dc-4f2c-994a-585b65893eee.png">